src/Http/Middleware/Authenticate.php

```diff
@@ -15,6 +16,10 @@ class Authenticate
      */
     public function handle($request, $next)
     {
-        return Horizon::check($request) ? $next($request) : abort(403);
+        if (! Horizon::check($request)) {
+            throw new UnauthorizedException(401);
```

Comment:
Any reason not to use `403` same as before?

Reply:
I probably pasted the incorrect exception on my last review. Sorry

Reply:
Well, the 403 status code !== unauthorized. We can use 403, but we should probably change the name of the exception to use "Forbidden" instead :)